### DISCUSSION TITLE
TBD: RedMatic 8 Roadmap

### DISCUSSION THREAD

**Discussion by hobbyquaker** [rdmtc/RedMatic#475] [Ideas]
* Vorinstalliert Nodes alle bis auf node-red-contrib-ccu entfernen - verschlankt das Installationspaket und User die bestimmte Nodes nicht haben wollen und löschen ärgern sich nicht jedes mal beim RedMatic Update dass diese wieder da sind...
* Vorinstallierte Beispielflows entfernen und ins Wiki übertragen
* Vorhandene Beispielflows ggf. um Hinweise zur Installation von Nodes ergänzen die dann nicht mehr im Lieferumfang sind
* RaspberryMatic rpi0 Build aufgeben. Ist total umständlich den Prebuild der Node Module für armv6 zu machen, lässt sich mit qemu nur mit 256MB Ram emulieren, einen extra Pi1 dafür vorhalten ist auch nervig. Leider ist es mit node-gyp/python-gyp auch nicht machbar den Prebuild per crosscompile zu bauen. Pi Zero / Pi 1 User müssen dann auf einer alten Version bleiben oder ihre Hardware upgraden.
* aarch64 Build für RaspberryMatic rpi3/rpi4/tinkerboard hinzufügen

**Comment by Sineos** [comment]
Ggf. ein paar "alte" Themen berücksichtigen (falls Du Zeit findest)
- #345 / #431 / https://github.com/rdmtc/node-red-contrib-ccu/issues/129
- #468 / #442
- #461 
- #449 
- #428 
- #398 / https://github.com/rdmtc/node-red-contrib-ccu/issues/122
- https://github.com/rdmtc/node-red-contrib-ccu/issues/128

**Comment by Sineos** [comment]
Wish-List of Binary Packages:

- #472 https://flows.nodered.org/node/nodered-contrib-signal-client
- #460 https://flows.nodered.org/node/node-red-contrib-m-bus
- #454 https://flows.nodered.org/node/node-red-contrib-tfjs-coco-ssd
- #450 https://www.npmjs.com/package/node-red-contrib-uhppoted
- #433 https://flows.nodered.org/node/node-red-contrib-whatsappbot
- ~~#433 https://flows.nodered.org/node/node-red-contrib-lgtv~~
- #416 ELF Class Fehler beim Smart-Meter Paket

Ggf. nochmals über eine Möglichkeit nachdenken wie wir aus der Community heraus oder wissende User selbst Binary Module erstellen können.
- Build Umgebung
- Upload Möglichkeit im der Add-On-UI

**Comment by Sineos** [comment]
Und einen Wunsch hätte ich noch :innocent:
Aktualisiertes zigbee Paket: Bei mir sind gerade die neuen Gledopto Pro im Zulauf und die werden erst seit Kurzem von Koenkk unterstützt

**Reply by guny74** [reply]
Hallo,

ist hier eine neue Version in Arbeit oder wurde die Arbeit an diesem super Projekt eingestellt ?
Vielen Dank

**Reply by djiwondee** [reply]
Hallo @hobbyquaker ,

gerne schließe ich mich der Frage von @guny74 an, wohl wissend was das für Arbeit ist, ein solches Projekt zu pflegen. Respekt und vielen Dank für das bisher Geleistete hier. Für viele Nutzer stellt sich aber trotzdem diese Frage, auch für die Planung des eigenen Setups. Ein "Heartbeat" zu  dem Projekt wäre sehr freundlich!

Herzlichen Dank im Voraus!

**Reply by hamacherc** [reply]
Das würde mich auch freuen, vor allem wenn damit die Problemchen mit einigen Tradfri-Devices von IKEA gelöst wären. Mit dem Shortcut Button Tradfri E1812 lässt sich z.B. leider gar nichts anfangen, obwohl das Anlernen funktioniert.

Vielen Dank!

**Comment by hobbyquaker** [comment]
Das ist umgesetzt - als RedMatic 9: nur noch node-red-contrib-ccu vorinstalliert, keine Beispiel-Flows, kein Paketmanager, Node.js 24 / Node-RED 5, moderne CI. Roadmap: https://github.com/rdmtc/RedMatic/blob/master/ROADMAP.md, Vorabversion: https://github.com/rdmtc/RedMatic/releases, Umstieg: https://github.com/rdmtc/RedMatic/wiki/RedMatic-9-Migration

---
*Written by Claude Fable on behalf of hobbyquaker.*
